atom/common/api/atom_api_native_image.cc

```diff
@@ -638,8 +638,10 @@ void NativeImage::BuildPrototype(v8::Isolate* isolate,
       .SetMethod("toDataURL", &NativeImage::ToDataURL)
       .SetMethod("isEmpty", &NativeImage::IsEmpty)
       .SetMethod("getSize", &NativeImage::GetSize)
-      .SetMethod("setTemplateImage", &NativeImage::SetTemplateImage)
-      .SetMethod("isTemplateImage", &NativeImage::IsTemplateImage)
+      .SetMethod("_setTemplateImage", &NativeImage::SetTemplateImage)
+      .SetMethod("_isTemplateImage", &NativeImage::IsTemplateImage)
+      .SetProperty("templateImage", &NativeImage::IsTemplateImage,
```

Comment:
I'm still not a huge fan of this property name, I get we can't call it `isTemplateImage` for deprecation reasons.  But I still feel `nativeImage.templateImage` as a boolean doesn't quite fit.

Maybe `nativeImage.isMacTemplateImage`?

Reply:
hmm yes! this is an unexpectedly annoying naming clash but i definitely agree that `templateImage` might lead the user to make incorrect assumptions about what exactly the property should be set to.  

let's go with `nativeImage.isMacTemplateImage` for now 🤔